src/lib/SX127xDriver/SX127x.cpp

```diff
@@ -14,7 +14,7 @@ void (*SX127xDriver::RXtimeout)() = &nullCallback;
 volatile WORD_ALIGNED_ATTR uint8_t SX127xDriver::TXdataBuffer[TXRXBuffSize] = {0};
 volatile WORD_ALIGNED_ATTR uint8_t SX127xDriver::RXdataBuffer[TXRXBuffSize] = {0};
 
-uint8_t SX127x_AllowedSyncwords[105] =
+const uint8_t SX127x_AllowedSyncwords[105] =
```

Comment:
Note: `const` array will be located into flash which is slower than RAM. `const` is good thing and just need to be forced into RAM. 
Not the issue with sync words since it these are called only during bootup...